src/com/palmergames/bukkit/towny/tasks/GatherResidentUUIDTask.java

```diff
@@ -79,4 +92,9 @@ private void applyUUID(Resident resident, UUID uuid, String source) {
 		TownySettings.incrementUUIDCount();
 		TownyMessaging.sendDebugMsg("UUID stored for " + resident.getName() + " received from " + source + ". Progress: " + TownySettings.getUUIDPercent() + ".");
 	}
+	
+	public static void setOfflineModeTrue() {

```

Comment:
Super super minor thing, but I feel like this naming is weird. Maybe something else simple like `markOfflineMode`. Idk, the current name is accurate, but something about it just feels very too literal.

Reply:
You make a good point.